extensions/epim/README.md

```diff
@@ -0,0 +1,43 @@
+# Entra PIM Role
+
+Activate Entra roles via PIM using Microsoft Graph API directly from Raycast.
+
+## Installation
+
+1. Ensure you have Raycast installed.
+2. Ensure that Powershell for macOS is installed.
```

Comment:
Your spelling of `PowerShell` is wrong. See https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/product-terminology?view=powershell-7.5.

Since PowerShell is a trademark, we should always use the correct spelling. Could you fix them?